src/data/complex/exponential.lean

```diff
@@ -884,6 +898,17 @@ by simp [sub_eq_add_neg, sinh_add, sinh_neg, cosh_neg]
 lemma cosh_sub : cosh (x - y) = cosh x * cosh y - sinh x * sinh y :=
 by simp [sub_eq_add_neg, cosh_add, sinh_neg, cosh_neg]
 
+/-- A real version of `complex.cosh_sq_sub_sinh_sq`-/
```

Comment:
I'm not sure this doc string is particularly helpful to be honest. It refers to a different lemma which isn't adjacent, and from the lemma statement it is clear what the lemma is.

Reply:
I want all my code to be documented, but I'm sure one docstring can't hurt